notebooks/Read_CoNLL-U_files.ipynb

```diff
@@ -0,0 +1,1116 @@
+{
```

Comment:
<p>It looks like the "head" columns ended up as floating point values somehow. They should be ints. Did some nulls sneak into the head pointers?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/198/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
I believe this actually has to do with 'virtual' tokens where words were omitted from the surface representation and had to be added by the labeller, which have numbers of  the form 10.1 or similar

Reply:
Do you have a suggestion of how I should handle this?

Reply:
The real root cause is upstream in Pandas. There is old code in various parts of Pandas that casts ints to floats for operations that *might* produce NA values. Pandas should be converting to nullable int types (see https://pandas.pydata.org/pandas-docs/stable/user_guide/integer_na.html) instead. I've been meaning to submit PRs to fix those problems but haven't had time.

For now, the best fix at the application level is to convert the series that Pandas has "floatified" back to an integer dtype. Use `int` if there aren't supposed to be NAs. Use `Int64` (with a capital "I") if there might be NAs.

Reply:
Follow-up: Looking some more at the code, I think the conversion to a nullable integer dtype should happen inside `conll_u_to_dataframes`.